src/components/EstimatedLifecycleCost/index.tsx

```diff
@@ -243,56 +247,46 @@ const EstimatedLifecycleCost = ({
         </div>
       </div>
       <div className="tablet:grid-col-7">
-        <div className="est-lifecycle-cost__year-costs margin-top-0">
-          <span className="text-bold">Year 1</span>
-          <Phase
-            formikKey={formikKey}
-            year={1}
-            values={years.year1}
-            setFieldValue={setFieldValue}
-            errors={errors.year1}
-          />
-        </div>
-        <div className="est-lifecycle-cost__year-costs">
-          <span className="text-bold">Year 2</span>
-          <Phase
-            formikKey={formikKey}
-            year={2}
-            values={years.year2}
-            setFieldValue={setFieldValue}
-            errors={errors.year2}
-          />
-        </div>
-        <div className="est-lifecycle-cost__year-costs">
-          <span className="text-bold">Year 3</span>
-          <Phase
-            formikKey={formikKey}
-            year={3}
-            values={years.year3}
-            setFieldValue={setFieldValue}
-            errors={errors.year3}
-          />
-        </div>
-        <div className="est-lifecycle-cost__year-costs">
-          <span className="text-bold">Year 4</span>
-          <Phase
-            formikKey={formikKey}
-            year={4}
-            values={years.year4}
-            setFieldValue={setFieldValue}
-            errors={errors.year4}
-          />
-        </div>
-        <div className="est-lifecycle-cost__year-costs">
-          <span className="text-bold">Year 5</span>
-          <Phase
-            formikKey={formikKey}
-            year={5}
-            values={years.year5}
-            setFieldValue={setFieldValue}
-            errors={errors.year5}
-          />
-        </div>
+        <Phase
+          formikKey={formikKey}
+          year={1}
```

Comment:
Why do we need this `year={1}` if we have the fiscal year?

I think this is still necessary in order to have the validations run properly. It'll be easily to have a fixed/known number to reference the year rather than a changing number. We might be able to remove this in a future iteration, maybe if/when we update the model for the estimated lifecycle costs.